.github/workflows/release.yml

```diff
@@ -9,50 +9,36 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with: { fetch-depth: 0 }
 
       - name: Set up Go
-        uses: actions/setup-go@v3
-        with: { go-version: 1.18 }
+        uses: actions/setup-go@v5
+        with: { go-version-file: go.mod }
 
       - name: Import GPG key
         id: import_gpg
-        uses: paultyng/ghaction-import-gpg@v2.1.0
```

Comment:
this package hasn't been maintained for 4 years and it used Node 12 (deprecated)

Reply:
Do we need a custom action for that? This could probably be achieved with a single `gpg` command.
I'm not saying we should fix that in this PR, but I'm not super comfortable using third-party actions for that kind of stuff. The surface for a supply chain attack is pretty high.

Reply:
yeah, it's possible by hand but it requires some magic parsing the outputs of the `gpg --list-secret-keys --keyid-format LONG` command..

from what I understand this GH Action is pretty popular and we use it in another places. if we're afraid we could maybe fork it or create our own version of it.

Reply:
yeah I mean it's not worse that what we had :D but we can discuss it for sure